chromadb/test/test_api.py

```diff
@@ -225,6 +225,22 @@ def test_add(client):
     assert collection.count() == 2
 
 
+def test_add_embeddings_without_ids(client):
```

Comment:
This is fine for now but I think in general we want to stop adding tests to this file; it's a giant kitchen sink and mostly a leftover of the initial tests we had at launch. I think we should instead have a test-per-api surface (not necessarily in this PR). I'll create an issue for that.

Reply:
Opened a new issue for this: https://github.com/chroma-core/chroma/issues/2586